packages/sdk/src/actor/behaviors/penBehavior.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { BehaviorType, ToolBehavior } from '.';
+import { TransformLike } from '../..';
+
+
+/* export */ interface DrawData {
+	transform: TransformLike;
+	// Potentially additional data to come, such as:
+	// pressure: number;
+}
+
+/* export */ interface PenEventData {
+    drawData: DrawData[];
+}
+
+/**
+ * Pen behavior class containing the target behavior actions.
+ */
+/* export */ class PenBehavior extends ToolBehavior<PenEventData> {
```

Comment:
Declaring the PenBehavior class but not exporting it will generate a lint warning (no-unused-vars).